src/applications/mhv/secure-messaging/components/Navigation.jsx

```diff
@@ -18,14 +20,40 @@ const Navigation = () => {
   const closeMenuButtonRef = useRef();
   const [navMenuButtonRef, setNavMenuButtonRef] = useState(null);
 
+  const initialPageTitle = convertPathNameToTitleCase(location.pathname);
+  const defaultInitialPageTitle = `${initialPageTitle} ${
+    PageTitles.PAGE_TITLE_TAG
+  }`;
+
+  const pageTitle = () => {
+    let generatedTitle = '';
+
+    if (initialPageTitle === 'Sent')
+      generatedTitle = `Sent messages ${PageTitles.PAGE_TITLE_TAG}`;
+    else if (initialPageTitle === 'New-message')
+      generatedTitle = `Start a new message ${PageTitles.PAGE_TITLE_TAG}`;
+    else generatedTitle = defaultInitialPageTitle;
+
+    return generatedTitle;
+  };
+
+  useEffect(
+    () => {
+      // initial page title
+      updatePageTitle(pageTitle());
+    },
+    [defaultInitialPageTitle, initialPageTitle, location.pathname],

```

Comment:
Same as a comment for `FolderHeader`. Sent can be handled in `FolderThreadListView` and Start a new message is more appropriate to a `Compose` container, what do you think?